.github/workflows/tests.yaml

```diff
@@ -55,10 +55,13 @@ jobs:
                       symfony-version: '4.4.*'
                     - package: 'serialization'
                       symfony-version: '4.4.*'
-                    # Exclude it is not support on PHP 8
+                    # Exclude as it is not support on PHP 8
                     # https://github.com/php-amqplib/php-amqplib/issues/846
                     - package: 'rabbitmq-bundle-bridge'
                       php-version: '8.0'
+                    # https://github.com/schmittjoh/JMSSerializerBundle/pull/819
+                    - package: 'jms-serializer-bundle-bridge'
+                      php-version: '8.0'
```

Comment:
This can be removed when https://github.com/schmittjoh/JMSSerializerBundle/pull/819 is merged.

Reply:
https://github.com/SimpleBus/SimpleBus/issues/92